tests/integration/test_charms.py

```diff
@@ -44,7 +44,7 @@ async def test_deploy_katib_charms(ops_test: OpsTest):
     # Deploy katib-controller, katib-db-manager, and katib-ui charms
     await ops_test.model.deploy(controller_charm, resources={"oci-image": controller_image_path})
```

Comment:
Do we need trust here as well?

Reply:
katib-controller is still a podspec so it's not needed